examples/connectivity/plot_extract_regions_smith_atlas_networks.py

```diff
@@ -0,0 +1,50 @@
+"""
+Regions Extraction of Default Mode Networks using Smith Atlas
+=============================================================
+
+This example shows how to extract regions which also includes
+default mode networks using ICA maps of Smith Atlas.
+
+"""
+# Fetch the datasets and atlas maps
+from nilearn import datasets
+from nilearn.image import iter_img
+from nilearn.image import new_img_like
+from nilearn.plotting import find_xyz_cut_coords
+print (" -- Fetching Smith Atlas ICA Networks -- ")
+smith_atlas = datasets.fetch_atlas_smith_2009()
+atlas_networks = smith_atlas.rsn10
+
+adhd_dataset = datasets.fetch_adhd(n_subjects=1)
+func_filenames = adhd_dataset.func
+
+# Region Extraction
+from nilearn.regions import region_extractor
+print (" -- Extracting Default Mode Networks -- ")
+extraction = region_extractor.RegionExtractor(atlas_networks, threshold=0.5,
+                                              thresholding_strategy='percentile',
+                                              extractor='local_regions',
+                                              min_size=200)
+extraction.fit_transform(func_filenames)
+
+regions = extraction.regions_
+# Index of each region to identify its corresponding ICA network
+index = extraction.identity_
+
+# Visualize the region extraction results
+import matplotlib.pyplot as plt
+from nilearn import plotting
+print (" -- Showing the region extraction results from a Smith RSN -- ")
+for i, cur_img in zip(index, iter_img(regions)):
+    coords = find_xyz_cut_coords(cur_img)
+    data = cur_img.get_data()
+    data.shape = (cur_img.get_data()).shape + (1, )
+    img = new_img_like(cur_img, data, cur_img.get_affine())
+    plotting.plot_prob_atlas(img, display_mode='z', cut_coords=coords[2:3],
+                             view_type='contours',
+                             title="Region extracted corresponds to Network of %d " % i)
```

Comment:
I am in a state such that atlas maps should not be shown as statistical maps. Correct me If I am wrong. Except that I am Ok to show as ```plot_stat_map```